src/go/rdctl/pkg/snapshot/snapshot.go

```diff
@@ -1,11 +1,27 @@
 package snapshot
 
 import (
+	"encoding/json"
 	"time"
 )
 
 type Snapshot struct {
-	Created time.Time
-	Name    string
+	Created time.Time `json:"created"`
+	Name    string    `json:"name"`
 	ID      string
```

Comment:
I think I would change `ID` to `id` if we are removing the capitals from the other names. I imagine you deliberately kept it as `ID` though, so :shrug:

Reply:
I've always preferred `ID` to `Id` as an abbrev for "identity", but might be in the minority, and definitely prefer to see it that way in a json payload.